docs/asset-store/docs/02-01-asset-store.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Asset Store
+type: Architecture
+---
+
+## Resources
+
+The whole concept of the Asset Store relies on these elements:
+
+- **Asset custom resource** is an obligatory [custom resource](./06-01-asset.md) (CR) in which you define the asset you want to store in a given storage bucket. Its definition requires the asset name and mode, the name of the Namespace in which it is available, the address to its Internet location, and the name of the bucket in which you want to store it. Optionally, you can specify the validation and mutation operations that the asset must undergo before it is stored.
+
+- **Asset Controller** validates if the bucket specified in the Asset CR exists, fetches the asset based on the information provided in the Asset CR, unpacks it if necessary, and performs optional asset validation and mutation. It finally uploads the asset into the specified Minio (Gateway) bucket.
+
+- **Bucket custom resource** is an obligatory [custom resource](./06-02-bucket.md) in which you define the name of the bucket for storing assets.
+
+- **Bucket Controller** listens to the events from the Bucket CR and checks if any new bucket was created or an existing one was deleted. It later uploads the new bucket onto Minio (Gateway) or deletes the bucket from it.
```

Comment:
`It later uploads the new bucket onto Minio` -> `It later creates the new  Minio (Gateway) bucket`? Because in the `Asset Controller` section we have: `It finally uploads the asset into the specified Minio (Gateway) bucket`, so this change would make sense to me.

Reply:
As discussed, I will shorten this part and make a reference to the lifecycle documents.